protocol/meson.build

```diff
@@ -32,6 +33,10 @@ protocols = [
 
 client_protocols = [
 	[wl_protocol_dir, 'unstable/xdg-shell/xdg-shell-unstable-v6.xml'],
+	[wl_protocol_dir, 'stable/xdg-shell/xdg-shell.xml'],
+	[wl_protocol_dir, 'unstable/idle-inhibit/idle-inhibit-unstable-v1.xml'],
+	'gamma-control.xml',
+	'gtk-primary-selection.xml',
```

Comment:
gamma-control and gtk-primary-selection are unused

Reply:
Did I add them? Then a rebase in between must have gone horribly wrong, since I certainly did not do this on purpose...